### PR TITLE
Add top companies for open source in China

### PR DESCRIPTION
**Pull request by AJLoveChina**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection

---------------------------------------------------------------------
### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Content (and my changes are in `index.md`)

> Add top companies for open source in China, they should be displayed in Github explore page.

### CODE REVIEW

**Comment by Lubbock1972** [comment]
Helping always work

File: collections/open-source-organizations/index.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ items:
  - OSGeo/osgeo
  - godaddy/godaddy.github.io
  - cloudflare/cloudflare.github.io
+ - eleme/eleme.github.io
+ - youzan/youzan
+ - didi/didi.github.io
+ - alibaba/alibaba.github.com
+ - baidu/baidu.github.io
+ - Tencent/tencent
```
Comment (kenyonj) [inline collections/open-source-organizations/index.md]: This doesn't seem to be a public repository.

**Comment by mgaf** [comment]
719604bd312479bd12d34cdcea769689394f5308

**Comment by mgaf** [comment]
719604bd312479bd12d34cdcea769689394f5308

**Comment by kenyonj** [comment]
https://github.com/adobe/adobe.github.com is a great example of a repo that explains an organizations stance on open source.

**Comment by kenyonj** [comment]
> I've gone through and looked at the rest of these entries that exist in this collection and I think we need to do some grooming. In light of that I don't see anything that is too relevant in any of these repos that you are adding that points to the open source efforts that any of these companies are making. Are there better repos that explain their efforts?

After a little more investigation I am going to reverse my stance on this.

Thanks again for your contribution!

**Pull request merged**